search/types_skipper.cpp

```diff
@@ -21,10 +21,8 @@ TypesSkipper::TypesSkipper()
   for (auto const & e : arrSkipEmptyName1)
     m_skipIfEmptyName[0].push_back(c.GetTypeByPath(e));
 
-  // Test for exact type (man_made-tower-communication is not).
   StringIL const arrSkipEmptyNameExact[] = {
     {"man_made", "chimney"},
-    {"man_made", "tower"},
```

Comment:
Just noticed here. IMO, there is no need to search for a regular unnamed "tower" by category search.

Reply:
By this logic we can potentially skip some other unnamed categories, incl. masts, water towers..

Reply:
What is displayed in Paris with this PR and "Eiffel Tower" query?